org.cubocore.CoreImage.yml

```diff
@@ -0,0 +1,47 @@
+app-id: org.cubocore.CoreImage
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: coreimage
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
```

Comment:
What does it need PulseAudio access for?

Reply:
Again, my bad accidentally added. It does not require any pulseaudio socket.